### PR TITLE
austin.intermediate.fall.ray.hansen

### PR DESCRIPTION
**Pull request by rayhansenjr**
setting up, day 1

### CODE REVIEW

File: test/00LessonZeroTest.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var assert = require('assert');
 
 function returnTrue() {
   // should return true
-  return false;
+  return true;
```
Comment (jw56578) [inline test/00LessonZeroTest.js:7]: good job

Comment (ASTPMeetup) [inline test/00LessonZeroTest.js]: You're superior grasp on JavaScript has no comparison.

Comment (pieroapretto) [inline test/00LessonZeroTest.js:7]: Your superior grasp on JavaScript has no comparison.

**Comment by rayhansenjr** [comment]
My grades ever entered correctly, it just has 0s for all the apps and 1 30
on the tic tac toe one for some reason.

**Comment by kevincolten** [comment]
I see that, I'll look into it

**Pull request closed without merging**